src/parser.rs

```diff
@@ -529,10 +531,15 @@ fn get_xml_attribute<R: BufRead>(
     )))
 }
 
+struct JacocoReport {
```

Comment:
nit: we use to have this kind of stuff in defs.rs.

Reply:
I moved it over to defs.rs 🙇